llms/googleai/vertex/vertex.go

```diff
@@ -229,6 +231,25 @@ func convertContent(content llms.MessageContent) (*genai.Content, error) {
 	return c, nil
 }
 
+// convertSystemContent merges the system message array in just one
+// after this function the system will send data in unique system instruction
+func convertSystemContent(c *genai.Content, content llms.MessageContent) (*genai.Content, error) {
+	var mergedSystemPrompt []string
+	for _, part := range content.Parts {
+		prompt, ok := part.(llms.TextContent)
+		if !ok {
+			return nil, ErrSystemJustSupportsText
+		}
+		mergedSystemPrompt = append(mergedSystemPrompt, prompt.Text)
+	}
+
+	c.Role = RoleSystem
+	merged := genai.Text(strings.Join(mergedSystemPrompt, ".\n"))
```

Comment:
Supporting multiple system prompts using splicing may have unexpected effects, so it is not recommended to modify them, and developers should be in control of their own system prompts